src/Sarif.Viewer.VisualStudio.UnitTests/SdkUiUtilitiesTests.cs

```diff
@@ -0,0 +1,197 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+using System;
+using System.Collections.Generic;
+using System.Windows;
+using System.Windows.Documents;
+using FluentAssertions;
+using Xunit;
+
+namespace Microsoft.Sarif.Viewer.VisualStudio.UnitTests
+{
+    public class SdkUiUtilitiesTests
+    {
+        private void Hyperlink_Click(object sender, RoutedEventArgs e) { }
+
+        [Fact]
+        public void SarifSnapshot_UnescapeBrackets()
+        {
+            string message = @"The quick \[brown fox\] jumps over the lazy dog.";
+            string actual = SdkUiUtilities.UnescapeBrackets(message);
+            actual.Should().Be("The quick [brown fox] jumps over the lazy dog.");
+        }
+
+        [Fact]
+        public void SarifSnapshot_GetMessageEmbeddedLinkInlines_DontCreateLinks()
+        {
+            string message = @"The quick [brown fox](2) jumps over the lazy dog.";
+
+            var expected = new List<Inline>
+            {
+                new Run("The quick "),
+                new Run("brown fox"),
+                new Run(" jumps over the lazy dog.")
+            };
+
+            var actual = SdkUiUtilities.GetInlinesForErrorMessage(message);
+
+            actual.Count.Should().Be(expected.Count);
+
+            actual[0].Should().BeOfType(expected[0].GetType());
+            (actual[0] as Run).Text.Should().Be((expected[0] as Run).Text);
+        }
```

Comment:
Loop over all of them, don't just check the first one.

Reply:
Oops copy/paste